packages/tabs/src/TabsContainer.spec.tsx

```diff
@@ -174,7 +178,7 @@ describe('TabsContainer', () => {
       expect(() => {
         render(
           <TabsContainer>
-            {({ getTabPanelProps }) => <div {...getTabPanelProps({ index: 0 })} />}
+            {({ getTabPanelProps }) => <div {...getTabPanelProps<any>({ index: 0 })} />}
```

Comment:
Why is the `<any>` necessary here? This should be inferred by the `useTabProps<any>` within `TabsContainer`

Reply:
The reason `<any>` is here is to suppress the TypeScript warning. I think TypeScript is unhappy that `item` is not provided: 

![Screen Shot 2019-10-09 at 2 54 07 PM](https://user-images.githubusercontent.com/1811365/66523373-aaa53280-eaa4-11e9-8b06-e6034393e1a8.png)

`item` is intentionally left off for this test which asserts that an error message is thrown.

Alternatively, I could make item optional on the `IGetTabPanelProps`  interface [here](https://github.com/zendeskgarden/react-containers/pull/111/files#diff-b36a30b996a4ee28a892a353beebe54bR23). Do you think that may be a better approach?

Reply:
Ahhh this makes sense. So this test is testing a runtime failure. TypeScript is stopping us from ever making that mistake (thanks TS ✋)

I would do `{ index: 0 } as any`. Since we are testing the JS use-case here that should be fine.

Reply:
Awesome, I updated this to use `as any`. Thank you for the suggestion!